src/publisher/api_v2_views.py

```diff
@@ -18,6 +16,12 @@
 LOG = logging.getLogger(__name__)
 
 
+class HttpResponse(DjangoHttpResponse):
+    @property
```

Comment:
can I change this a writeable setter? logic in publisher/middleware.py might benefit

Reply:
yes, it's a bit weird: https://pythonguide.readthedocs.io/en/latest/python/property.html#property